ProjFS.Mac/PrjFSKextTests/HandleOperationTests.mm

```diff
@@ -116,6 +116,8 @@ - (void) testEmptyFileHydrates {
         KAUTH_VNODE_WRITE_DATA,
         KAUTH_VNODE_EXECUTE,
         KAUTH_VNODE_DELETE,
+        KAUTH_VNODE_APPEND_DATA,
+        KAUTH_VNODE_ADD_SUBDIRECTORY,
```

Comment:
We shouldn't receive `KAUTH_VNODE_ADD_SUBDIRECTORY` for file vnodes.  Should we only be adding `KAUTH_VNODE_APPEND_DATA` here?